packages/editor/src/store/reducer.native.js

```diff
@@ -80,6 +80,25 @@ export function notices( state = [], action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the block insertion event list state.
+ *
+ * @param {Object}  state  Current state.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function lastBlockInserted( state = {}, action ) {
+	switch ( action.type ) {
+		case 'ADD_LAST_BLOCK_INSERTED':
+			return { ...state, clientId: action.clientId };
+
+		case 'CLEAR_LAST_BLOCK_INSERTED':
+			return {};
+	}
+	return state;
+}
```

Comment:
Not a blocker at all but I'm curious if state could have been just a plain `clientId` (which is a string or an int IIRC):
```
export function lastBlockInserted( state = null, action ) {
	switch ( action.type ) {
		case 'ADD_LAST_BLOCK_INSERTED':
			return action.clientId;

		case 'CLEAR_LAST_BLOCK_INSERTED':
			return null;
	}
	return state;
}
```
I guess that would have the disadvantage of lastBlockInserted not being auto-documenting (like it doesn't say it's a clientId anymore). Probably keeping it an object is also more scalable since it's easy to add more attributes without having to refactor.